scripts/configure-hypervisor.sh

```diff
@@ -92,10 +92,15 @@ check_tag()
 	local tag="$1"
 	local entry="$2"
 
+	[ -z "$tag" ] && die "no tag for entry '$entry'"
+	[ -z "$entry" ] && echo -e "\n" && die "no entry for tag '$tag'"
```

Comment:
why `echo -e "\n"` ?

Reply:
Yeah, taking another look, this is just plain ugly so I've removed those changes.